src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -297,7 +345,7 @@ else if (child.getType() == JavadocTokenTypes.JAVADOC_TAG) {
     /**
      * Concatenates string within text of html tags.
      *
-     * @param result javadoc string
+     * @param result     javadoc string
```

Comment:
revert all unrelated changes, please fix your IDE to avoid formatting.

Reply:
done